CONTRIBUTORS.md

```diff
@@ -0,0 +1,3 @@
+# Contributors
+### This open source project was made with ❤️ by 1 users!
+* [jwu910](https://github.com/jwu910) - 2
```

Comment:
wow how rude, only including yourself ... 🤔

Reply:
Reads from master lol... maybe need to read from current branch instead. Otherwise this is always gonna be behind when a user pushes to master

Reply:
Yeah... we need to figure that out... haha. Let's throw a todo in there and figure out if this is a limitation with the API or if we're missing something else.